readthedocs/api/v2/utils.py

```diff
@@ -20,8 +20,20 @@
 log = logging.getLogger(__name__)
 
 
-def sync_versions(project, versions, type):  # pylint: disable=redefined-builtin
-    """Update the database with the current versions from the repository."""
+def sync_versions_db(project, versions, type):  # pylint: disable=redefined-builtin
```

Comment:
It feels weird this name, what about `sync_versions_to_db` and `delete_versions_from_db`?